code/game/machinery/cloning.dm

```diff
@@ -109,7 +111,27 @@ GLOBAL_VAR_INIT(clones, 0)
 	. = ..()
 	. += "The write-protect tab is set to [read_only ? "protected" : "unprotected"]."
 
+// Biomass
 
+/obj/machinery/clonepod/attackby(obj/item/W, mob/user, params)
+	var/old_biomass = biomass // saves biomas variable
+	if(istype(W, /obj/item))
+		if(istype(W, /obj/item/stack/sheet/animalhide/human)) // lets play flesh or meat
+			biomass += 50
+		if(istype(W, /obj/item/reagent_containers/food/snacks/meat/slab)) // Are we inserting meat?
```

Comment:
Make this an else if, add a else block with just `return` and remove the weird if biomass != old_biomass thing to prevent badcode that assumes things